### DISCUSSION TITLE
Last update custom FBW

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#4632] [General]
Hi developers,

is that the actual version of the experimental mod? I am not sure if my installer is not working proberly, cause round about 3 days no newer version?!

![grafik](https://user-images.githubusercontent.com/76531893/116420094-f6adf880-a83d-11eb-981c-d928082a9c51.png)

Cheers

**Comment by Kamegohan** [comment]
Hi, I am following FBW on Twitter and there they announced, that experimental version will get copied to dev version 1:1. So if you used the custom FBW version you can now switch to dev version and will get further updates to all the (at this point) implemented features in future.

From this point the experimental FBW version will get more custom features which are far more experimental than earlier builds features.

You can read by yourself under the following link:

https://twitter.com/FlyByWireSim/status/1385715201853595648?s=19

**Comment by ExampleWasTaken** [comment]
The experimental version is no longer updated and by now already behind the development version. It will receive updates again once the next feature moves into the experimental phase. Please use either the stable or the development version for now.

EDIT: Please make sure you post in the right category! :)

**Reply by CptBayer** [reply]
Ah ok :-) .... Thanx for the info... 

Greets klaus
